tools/internal_ci/macos/grpc_interop_toprod.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+# change to grpc repo root
+cd $(dirname $0)/../../..
+
+source tools/internal_ci/helper_scripts/prepare_build_macos_interop_rc
+source tools/internal_ci/helper_scripts/prepare_build_macos_rc
+
+# using run_interop_tests.py without --use_docker, so we need to build first
+tools/run_tests/run_tests.py -l c++ -c opt --build_only
+
+export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH="$(pwd)/etc/roots.pem"
+
+# NOTE: only tests a subset of languages for time & dependency constraints
+# building all languages in the same working copy can also lead to conflicts
+# due to different compilation flags
+# TODO(jtattermusch): add --cloud_to_prod_auth support
+tools/run_tests/run_interop_tests.py -l c++ --cloud_to_prod -t -j 4
```

Comment:
If we are run against only 1 backend, let's use --prod_servers cloud_gateway_v4, which is the Cloudpath and use grpc all the way to backends.

Reply:
As this setup is experimental for now, I'll leave as is for now and we can discuss offline what it the best gateway to use (to be honest, i thought cloud gateway doesn't make sense when outside of GCP, in which case gateway_v4 would probably make sense - we can discuss offline).